BookReader/BookReader.js

```diff
@@ -2952,7 +2952,7 @@ BookReader.prototype.search = function(term, options) {
     //url    += '/fulltext/inside.php?item_id='+this.bookId;
     url    += '/fulltext/new_inside.php?item_id='+this.bookId;
     url    += '&doc='+this.subPrefix;   //TODO: test with subitem
-    url    += '&path='+this.bookPath.replace(new RegExp('/'+this.subPrefix+'$'), ''); //remove subPrefix from end of path
+    url    += '&path=' + this.bookPath.replace('/' + this.subPrefix, ''); //remove subPrefix from end of path
```

Comment:
This is a good step forward, but I think you need replace the last instance. For example:

```
FooBarFoo
```

Should become

```
FooBar
```

And it should only replace at the end of the string.

So 

```
FooBarFoo2
```
Should not get replaced:
```
FooBarFoo2
```

It should be this way, because the original regex used `$` at the end.